jihyeon/models/userDao.js

```diff
@@ -27,6 +27,25 @@ const createUser = async (email, password, name, age, phoneNumber) => {
   }
 };
 
+const login = async (email) => {
```

Comment:
```suggestion
const getUserByEmail = async (email) => {
```
dao는 Data Access Object의 약자로서 해당 부분에 작성되는 함수이름으로 login은 적합하지 않습니다. CRUD(create, get, update, delete)와 자원의 조합으로 getUserByEmail과 같은 식으로 작성해주세요.